src/components/Button/Button.js

```diff
@@ -55,8 +57,9 @@ Button.propTypes = {
   name: PropTypes.string,
   corners: PropTypes.oneOf(['square', 'round', 'circle']),
   disabled: PropTypes.bool,
-  style: PropTypes.object,
+  style: PropTypes.string,
   type: PropTypes.oneOf(['default', 'link', 'outline']),
```

Comment:
we also need to change this type PropTypes, or remove it really, can just leave type to be within the ...extraProps and have it hadle itself